internal/uvm/vpmem.go

```diff
@@ -41,19 +41,28 @@ func newDefaultVPMemInfo(hostPath, uvmPath string) *vPMemInfoDefault {
 	}
 }
 
+// fileSystemSize retrieves ext4 fs SuperBlock and returns the file system size and block size
+func fileSystemSize(vhdPath string) (uint64, int, error) {
+	sb, err := tar2ext4.ReadExt4SuperBlock(vhdPath)
+	if err != nil {
+		return 0, 0, errors.Wrap(err, "failed to read ext4 super block")
+	}
+	blockSize := 1024 * (1 << sb.LogBlockSize)
+	fsSize := uint64(blockSize) * uint64(sb.BlocksCountLow)
```

Comment:
I actually moved this function from a `vpmem_mapped.go`, which had this same signature. But since the only code path it's used is in `readVeritySuperBlock`, I can probably just cast to  `int64` here and avoid recasting to `int64` when making `DeviceVerityInfo`